lib/hanami/components/components.rb

```diff
@@ -28,7 +28,13 @@ module Components # rubocop:disable Metrics/ModuleLength
       end
 
       resolve do |configuration|
-        Hanami::Logger.new(Hanami.environment.project_name, *configuration.logger) unless configuration.logger.nil?
+        if configuration.logger.is_a?(Array)
+          if configuration.logger.length == 1 && configuration.logger[0].is_a?(::Logger)
```

Comment:
and are you sure that this code will work with other loggers (for example [logging](https://github.com/TwP/logging))?